cmd/kubectl-gadget/trace/oomkill.go

```diff
@@ -15,107 +15,39 @@
 package trace
 
 import (
-	"fmt"
-	"strings"
-
 	"github.com/spf13/cobra"
 
 	commontrace "github.com/kinvolk/inspektor-gadget/cmd/common/trace"
 	commonutils "github.com/kinvolk/inspektor-gadget/cmd/common/utils"
 	"github.com/kinvolk/inspektor-gadget/cmd/kubectl-gadget/utils"
-	"github.com/kinvolk/inspektor-gadget/pkg/gadgets/trace/oomkill/types"
+	"github.com/kinvolk/inspektor-gadget/pkg/gadgets/trace/fsslower/types"
+	oomkillTypes "github.com/kinvolk/inspektor-gadget/pkg/gadgets/trace/oomkill/types"
 )
 
 type OOMKillParser struct {
 	commonutils.BaseParser[types.Event]
 }
 
```

Comment:
Not needed anymore.

Reply:
That is correct. And VSCode imported `fsslower/types` automagically because of `types.Event`